lib/screen/global_receive/receive_detail_page.dart

```diff
@@ -69,68 +70,81 @@ class _GlobalReceiveDetailPageState extends State<GlobalReceiveDetailPage> {
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: [
+                      Text(
+                        'important'.tr(),
+                        style: theme.textTheme.ppMori700Black14,
                       ),
-                      color: Color.fromRGBO(237, 237, 237, 0.3),
-                    ),
-                    child: Column(
-                        crossAxisAlignment: CrossAxisAlignment.stretch,
-                        children: [
-                          Text(
-                            "account_address".tr(),
-                            textAlign: TextAlign.left,
-                            style: ResponsiveLayout.isMobile
-                                ? theme.textTheme.atlasGreyBold12
-                                : theme.textTheme.atlasGreyBold14,
-                          ),
-                          const SizedBox(height: 4.0),
-                          Padding(
-                            padding: const EdgeInsets.only(left: 8),
-                            child: Text(
-                              _account.accountNumber,
-                              textAlign: TextAlign.start,
-                              softWrap: true,
-                              style: theme.textTheme.titleSmall,
-                            ),
-                          ),
-                        ]),
-                  )),
-              SizedBox(
-                  height: 25,
-                  child: Container(
-                      alignment: Alignment.center,
-                      child: _copied
-                          ? Text(
-                              "copied".tr(),
-                              style: ResponsiveLayout.isMobile
-                                  ? theme.textTheme.atlasBlackBold12
-                                  : theme.textTheme.atlasBlackBold14,
-                            )
-                          : const SizedBox())),
-              const SizedBox(height: 4),
-              Text(
-                _blockchainWarningText(_account.blockchain),
-                style: ResponsiveLayout.isMobile
-                    ? theme.textTheme.atlasGreyNormal12
-                    : theme.textTheme.atlasGreyNormal14,
+                      const SizedBox(height: 15),
+                      Text(_blockchainWarningText(_account.blockchain),
+                          style: theme.textTheme.ppMori400Black14),
+                    ],
+                  ),
+                ),
+              ),
+              const SizedBox(height: 16),
+              Container(
+                decoration: BoxDecoration(
+                  color: AppColor.auLightGrey,
+                  borderRadius: BorderRadiusGeometry.lerp(
+                      const BorderRadius.all(Radius.circular(5)),
+                      const BorderRadius.all(Radius.circular(5)),
+                      5),
+                ),
+                child: Padding(
```

Comment:
the container has `padding` already, so we don't need to wrap Column inside a padding.